bubble-wrap-style.yaml

```diff
@@ -5392,28 +5411,125 @@ layers:
                             style: italic
                             stroke: { color: white, width: 3px }
 
-#    transit-bus-roads:
-#        data: { source: mapzen, layer: roads }
-#        filter: { is_bus_route: true }
-#        draw:
-#            lines:
-#                visible: global.sdk_transit_overlay
-#                interactive: false
-#                color: [[13,[0,0,1,0.5]],[15,blue]]
-#                width: [[11,0.5px],[12,0.8px],[16,1.25px],[18,1m]]
-#                # let roads sort themselves past zoom 14
-#                order: 488
-#                # but give them all the same outline
-#                outline:
-#                    order: 487
+    transit-bus-roads:
+        data: { source: mapzen, layer: roads }
+        filter: { is_bus_route: true }
+        draw:
+            lines:
+                visible: global.sdk_transit_overlay
+                interactive: false
+                color: [[13,[0,0,0.5,0.25]],[15,[0,0,0.5,0.8]]]
+                width: [[11,0.5px],[12,0.8px],[16,1.5px],[18,1.5m]]
+                # let roads sort themselves past zoom 14
```

Comment:
This is just assigning an explicit order value though, is the comment incorrect? (or the code?)

Reply:
Updated the comment in 1da2c657047b2b9d6ba99306a5fa0ee6010f466e.

We moved the zoom changes server side a few releases ago so the comment was outdated. There look to be a few static int values in the scene file still. I've filed https://github.com/tangrams/bubble-wrap/issues/278 to track that.